test/functoria/query/dune.full.expected

```diff
@@ -0,0 +1,33 @@
+;; Generated by "test query dune".
+
+(executable
+  (name config)
+  (flags (:standard -warn-error -A))
+  (modules config)
+  (libraries f0 functoria))
+
+(rule
+  (alias dune)
+  (targets dune)
+  (mode promote-until-clean)
+  (action (with-stdout-to dune (run ./config.exe query dune))))
+
+(rule
+  (alias configure)
+  (targets key_gen.ml main.ml noop.install noop.opam)
+  (action (run ./config.exe configure)))
+
+(rule
+  (alias build)
+  (targets vote warn_error)
+  (action (run ./config.exe build)))
+
+(rule
+  (alias clean)
+  (action (run ./config.exe clean)))
+
+(executable
+   (name      main)
+   (modules   (:standard \ config))
+ ; (promote   (until-clean))
```

Comment:
Why do we need a commented line?

Reply:
If I uncomment this line, the rule is evaluated unconditionally for some reason. Will try to make a smaller repro when everything works.